docker-build-config/wait-for

```diff
@@ -1,5 +1,27 @@
 #!/bin/sh
 
+# The MIT License (MIT)
```

Comment:
Just checked the [policy](https://www.apache.org/legal/src-headers.html#3party) of ASF license header of the third party, we don't have right the change the license header if we are not the copyright owner or owner's agent. I think we just need to list the file location in the License file for source release.

Reply:
So I should just revert the change, right? And root/LICENSE is already list this script so it's nothing should be done with root/LICENSE, right?